src/libs/utils.js

```diff
@@ -1,7 +1,7 @@
 import Noty from 'noty';
 import 'noty/lib/noty.css';
 import 'noty/lib/themes/bootstrap-v3.css';
-import {map as nonFPMap} from 'lodash';
+import {map as nonFPMap, forEach as nonFPForEach} from 'lodash';
```

Comment:
So there's two forEach's being used in this file, one from 'lodash/fp' and one from 'lodash'. The reason being is that with the fp version, the iterating function is `(value) => //code here` where as in the regular version its `(value, key) => //insert code here`.

As for why we need two versions, remember the error Shae pointed out here: https://github.com/DataBiosphere/duos-ui/pull/1718#pullrequestreview-1067388069

It's being caused by this function in  `utils.js`

`export const applyHoverEffects = (e, style) => {
  nonFPForEach((key, value) => {
    e.target.style[key] = value;
  })(style);
};`

What had happened was that the normal `forEach` that was used here was replaced with the `fp` variant. The difference in the iterating function causes the function to error out. 

So in short, we need both functions, but they can't be named the same variable, hence the `nonFPForEach`.

Reply:
Ok! Didnt realize it was because both lodash and lodash/fp were beign used.... That being said, I still think the name is confusing (just because it wasn't immediately obvious to me what it was "nonFP" was supposed to mean. Don't know what to do about that though

Reply:
maybe it could be `lodashForEach` and `lodashFpForEach` ? that makes both mirror each other.

Reply:
maybe it could be `lodashForEach` and `lodashFpForEach` ? that way they mirror each other

Reply:
Another way to solve this is to import a prefix for either the non-fp or the fp version. In the past, I've used `import _ from 'lodash';` and `import * as fp from 'lodash/fp';`, for example

Reply:
Yeah but that would cause Webpack to import the entire `lodash` library on file bundling, and lodash is a pretty big library. Even if the code is minified, I'd rather not import methods that are never used for a particular file.

Reply:
> maybe it could be `lodashForEach` and `lodashFpForEach` ? that way they mirror each other

That works for me